packages/cdk8s-plus/README.md

```diff
@@ -656,3 +655,54 @@ const awsService = new kplus.ServiceAccount(chart, 'AWS');
 // give access to the aws creds secret.
 awsService.addSecret(awsCreds);
 ```
+
+### `Ingress`
+
+[Ingress] manages external access to services in a cluster, typically through
+HTTP. Ingress may provide load balancing, SSL termination and name-based virtual
+hosting.
+
+You must have an [Ingress controller] to satisfy an Ingress. Only creating an
+Ingress resource has no effect.
+
+> API Reference: [Ingress](./API.md#cdk8s-plus-ingress)
+
+[Ingress]: https://kubernetes.io/docs/concepts/services-networking/ingress/
+[Ingress controller]: https://kubernetes.io/docs/concepts/services-networking/ingress-controllers
+
+The following example will route HTTP requests sent to the `/hello` url prefix
+to a service associated with a deployment of the
+[hashicorp/http-echo](https://github.com/hashicorp/http-echo) image.
+
+```ts
+const helloDeployment = new kplus.Deployment(this, text, {
+  spec: {
+    podSpecTemplate: {
+      containers: [
+        new kplus.Container({
+          image: 'hashicorp/http-echo',
+          args: [ '-text', 'hello ingress' ]
+        })
+      ]
+    }
+  }
+});
+
+const helloService = helloDeployment.expose({ port: 5678 });
+
+const ingress = new Ingress(this, 'ingress');
+ingress.addRule('/hello', kplus.IngressBackend.fromService(helloService));
+```
+
+Yo can use `addHostRule(host, path, backend)` to define a route that will only
+apply to requests sent to a specific host name. This can be used to implement
```

Comment:
Does the `host` refer to request target or origin? Because the `addHostDefaultBackend` docs say: 

```ts
   * Specify a default backend for a specific host name. This backend will be used as a catch-all for requests
   * originating from the specified host name.
```

Would be nice to add some request examples.

Reply:
The target. I'll fix the doc

Reply:
fixed